dune

```diff
@@ -8,3 +8,8 @@
  (release
   (flags
    (:standard -g -w -18-53))))
+
+(rule
+ (alias bench)
+ (action
+  (diff driver-benchmarks.json doc/driver-benchmarks.json)))
```

Comment:
This creates an error when `driver-benchmarks.json` is not already present at the root of the project (which is the case in the repo)... maybe generate an empty one if there is none?